README.md

```diff
@@ -1,306 +1,61 @@
-
-You can opt-out MAC assignment on your namespace by adding the following labels:
-- `mutatepods.kubemacpool.io=ignore` - to opt-out MAC assignment for Pods in your namespace
-- `mutatevirtualmachines.kubemacpool.io=ignore` - to opt-out MAC assignment for VMs in your namespace
+Check the under interfaces section:
 
-To disable kubemacpool MAC assignment on a specific namespace:
 ```bash
-kubectl label namespace example-namespace mutatepods.kubemacpool.io=ignore mutatevirtualmachines.kubemacpool.io=ignore
-namespace/example-namespace labeled
-```
-
-To re-enable kubemacpool MAC assignment in a namespace:
-```bash
-kubectl label namespace example-namespace mutatepods.kubemacpool.io- mutatevirtualmachines.kubemacpool.io-
-namespace/example-namespace labeled
-```
-
-#### How to opt-in kubemacpool MAC assignment for a namespace in Opt-in mode
-
-You can opt-in kubemacpool MAC assignment on your namespace by adding the following labels:
-- `mutatepods.kubemacpool.io=allocate` - to opt-in MAC assignment for Pods in your namespace
-- `mutatevirtualmachines.kubemacpool.io=allocate` - to opt-in MAC assignment for VMs in your namespace
-
-To disable kubemacpool MAC assignment on a specific namespace:
-```bash
-kubectl label namespace example-namespace mutatepods.kubemacpool.io- mutatevirtualmachines.kubemacpool.io-
-namespace/example-namespace labeled
-```
-
-To re-enable kubemacpool MAC assignment in a namespace:
-```bash
-kubectl label namespace example-namespace mutatepods.kubemacpool.io=allocate mutatevirtualmachines.kubemacpool.io=allocate
-namespace/example-namespace labeled
-```
-
-**note:** If a VMI is created directly and not through a VM, then it is handled in kubemacpool by the pod handler.
-
-### Check deployment
-
-Configmap:
-```bash
-[root]# kubectl -n kubemacpool-system describe configmaps
-
-Name:         kubemacpool-mac-range-config
-Namespace:    kubemacpool-system
-Data
-====
-RANGE_END:
-----
-FD:FF:FF:FF:FF:FE
-
-RANGE_START:
-----
-02:00:00:00:00:11
-```
-
-pods:
-```bash
-kubectl -n kubemacpool-system get po
-NAME                                                  READY   STATUS    RESTARTS   AGE
-kubemacpool-mac-controller-manager-6894f7785d-t6hf4   1/1     Running   0          107s
-```
-
-### Example
-
-Create a network-attachment-definition:
-
-The 'NetworkAttachmentDefinition' is used to setup the network attachment, i.e. secondary interface for the pod.
-This is follows the [Kubernetes Network Custom Resource Definition De-facto Standard](https://docs.google.com/document/d/1Ny03h6IDVy_e_vmElOqR7UdTPAG_RNydhVE1Kx54kFQ/edit)
-to provide a standardized method by which to specify the configurations for additional network interfaces.
-This standard is put forward by the Kubernetes [Network Plumbing Working Group](https://docs.google.com/document/d/1oE93V3SgOGWJ4O1zeD1UmpeToa0ZiiO6LqRAmZBPFWM/edit).
-
-```yaml
-apiVersion: "k8s.cni.cncf.io/v1"
-kind: NetworkAttachmentDefinition
+apiVersion: kubevirt.io/v1alpha3
```

Comment:
Could you use `yaml` instead of `bash`? (```yaml)

Reply:
right. Done